### PR TITLE
#37native server

### PR DESCRIPTION
**Pull request by ryanemerson**
https://github.com/infinispan/infinispan-images/issues/37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
-# Infinispan Server Image
+# Infinispan Server Images
 
 This repository contains various artifacts to create Infinispan server images.
 
+## Images
+Currently we provide the following images which are all based upon the [ubi-minimal](https://catalog.redhat.com/software/containers/detail/5c359a62bed8bd75a2c3fba8)
+base image:
+
+- `infinispan/server` - Infinispan is executed using the Java 11 openjdk jvm
```
Comment (tristantarrant) [inline README.md:9]: JVM is all-caps

File: README.md [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+- `infinispan/server-native` - Infinispan is executed natively using the [Infinispan Quarkus](https://github.com/infinispan/infinispan-quarkus) binary.
+
+> The `server` and `server-native` images are configured the same. The instructions throughout this docs are applicable
```
Comment (tristantarrant) [inline README.md:12]: this doc or these docs

File: server-native.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  - value: 7800
+  - value: 11221
+  - value: 11222
+  - value: 45700
+  - value: 57600
+  labels:
+  - name: name
+    value: Infinispan Server
+  - name: version
+    value: 11.0.0.Dev04
```
Comment (tristantarrant) [inline server-native.yaml:43]: Shouldn't this be Dev05 ?

File: server-openjdk.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- name: infinispan/server
+  version: 11.0.0-SNAPSHOT
+  description: Infinispan Server
+  from: registry.access.redhat.com/ubi8/ubi-minimal:8.2-267
+  artifacts:
+  - name: config-generator
+    image: native-builder
+    path: /opt/config-generator
+  - name: server
+    url: https://downloads.jboss.org/infinispan/11.0.0.Dev04/infinispan-server-11.0.0.Dev04.zip
```
Comment (tristantarrant) [inline server-openjdk.yaml:24]: Can't we use some form of variable substitution to avoid hardcoding the version number in many places ?

Comment (ryanemerson) [reply server-openjdk.yaml:24]: Cekit doesn't allow variable substitution in the yaml, instead it allows yaml nodes to be [overridden](https://docs.cekit.io/en/latest/handbook/overrides.html).

However the creation of these URLs are automated by `Jenkinsfile-release` anyway, so it's simply case of entering the Infinispan version in the build and the descriptor file is updated accordingly. The `Jenkinsfile-release` hasn't been updated to reflect the fact we have multiple images, but that's my plan for today.

File: server-native.yaml (hunk 3 above)
Comment (ryanemerson) [reply server-native.yaml:43]: Yep just copy and pasted from the old `image.yaml`, I'll fix.

**Pull request merged**